src/app/search/query.js

```diff
@@ -25,54 +26,70 @@ const addFieldQuery = (queryString, field = 'keyword') => {
    * Strip punctuation and process spaces as plus signs for final split.
    * ES characters to escape before sending: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /
    */
-  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\$&').trim().replace(/\s+/g, '+').split('+');
+  /**
+  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\$&').trim()
```

Comment:
You can remove this if you're not using it.

Reply:
cleaned